crates/dojo-indexer/src/hash.rs

```diff
@@ -0,0 +1,11 @@
+use prisma_client_rust::bigdecimal::num_bigint::BigUint;
+use sha3::{Keccak256, Digest};
+
+pub fn starknet_hash(data: &[u8]) -> BigUint {
```

Comment:
hm maybe it would be better to use this: https://github.com/starkware-libs/cairo/blob/abd35afa559170cfe7d283ca7c4b9bb833783a05/crates/cairo-lang-starknet/src/contract.rs#L32